src/components/ConversationSettings/ConversationSettingsDialog.vue

```diff
@@ -30,6 +30,15 @@
 				<p v-if="recordingConsentRequired">
 					{{ t('spreed', 'The consent to be recorded will be required for each participant before joining every call.') }}
 				</p>
+				<NcCheckboxRadioSwitch v-if="supportsArchive"
+					type="switch"
+					:checked="isArchived"
+					@update:checked="toggleArchiveConversation">
+					{{ t('spreed', 'Archive conversation') }}
+				</NcCheckboxRadioSwitch>
+				<p>
```

Comment:
Should by styled like `conversation-permissions-editor__hint` is on the same modal